src/Tasks/Resources/Strings.resx

```diff
@@ -1127,6 +1127,10 @@
     <value>MSB3822: Preserialized resources require the System.Resources.Extensions assembly at runtime, but it was not found in this project's references.</value>
     <comment>{StrBegin="MSB3822: "}</comment>
   </data>
+  <data name="GenerateResource.PreserializedResourcesRequiresProperty">
+    <value>MSB3823: Preserialized resources require the property GenerateResourceUsePreserializedResources to be set to true.</value>
```

Comment:
Are people going to understand what these messages mean?  Are they going to know that they are using preserialized resources?

Reply:
Also pre-existing (elsewhere), but fixed in d5b6ccf